src/config/config.go

```diff
@@ -0,0 +1,15 @@
+package config
```

Comment:
👏

Reply:
i needed to import this struct in other packages.